task_1/plotting_utils/plot_distributions.py

```diff
@@ -0,0 +1,63 @@
+import os
```

Comment:
The notebook is a scratch pad that I used for writing the function. The intention is to have all finalised plotting utilities in a single place (plotting_utils/). Not split out across  notebooks. 

I think it is better to remove it from the notebook and instead import the function from plotting_utils.py

Reply:
The function takes in any two samples (is general). It can be used for any data partition not just GEX. I just provided an example.

Reply:
> The function takes in any two samples (is general). It can be used for any data partition not just GEX. I just provided an example.

yeah I didn't look too much into it I guess, but a few more plots would be nice to get a better view of the dataset :)